source/georef/georef.cpp

```diff
@@ -863,6 +866,12 @@ bool GeoRef::add_parking_edges(const type::GeographicalCoord& coord) {
     return true;
 }
 
+void GeoRef::init_inversed_nb_vertex_by_mode() {
+    // a math trick to handle the round error:
+    // use (N + 0.1)/N^2 instead of 1/N
```

Comment:
Out of curiosity, do you have any resource that explains that ?

Reply:
I found that myself...

Reply:
and tested on my machine...

Reply:
comments added